src/main/java/ch/njol/skript/bukkitutil/HealthUtils.java

```diff
@@ -91,8 +87,17 @@ public static void damage(Damageable e, double d) {
 		e.damage(d * 2);
 	}
 
+	public static void damage(Damageable e, double d, DamageSource cause) {
+		if (d < 0) {
```

Comment:
Didn't realize this but we need proper naming of these values `e`, `d` don't explain anything `e` could be `event`, `d` could be `damageable`

Since we're doing it here we might as well apply these to everything in the class rather than just here